packages/wallet/ui/src/components/Offer.jsx

```diff
@@ -132,25 +140,32 @@ const OfferWithoutContext = ({
     );
   };
 
+  let feeValueString = 'unknown';
+  try {
+    feeValueString =
+      fee &&
+      stringifyPurseValue({
+        value: fee.value,
+        displayInfo: fee.displayInfo,
+      });
+  } catch {
+    console.warn("Couldn't parse fee value", fee);
+  }
```

Comment:
isn't this also an amount? why repeat this logic?

if it's for data validation that should happen on the way into application state.